### PR TITLE
Tweak to allow multiple conditions to combine results

### PR DESCRIPTION
**Pull request by jsight**

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/175/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/175/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport'>org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/175/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport/org.jboss.windup.qs.rules/EjbBeanUtilsAsyncUsageRuleProviderTest/testRules/'><strong>org.jboss.windup.qs.rules.EjbBeanUtilsAsyncUsageRuleProviderTest.testRules</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/809/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.rules.apps:windup-rules-xml-tests' /><a href='http://10.3.8.5/job/windup-pr-builder/809/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport'>org.jboss.windup.rules.apps:windup-rules-xml-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-pr-builder/809/org.jboss.windup.rules.apps$windup-rules-xml-tests/testReport/org.jboss.windup.rules.xml/XmlFileParameterizedTest/testXmlParamsMultiCondition/'><strong>org.jboss.windup.rules.xml.XmlFileParameterizedTest.testXmlParamsMultiCondition</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/810/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/176/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/176/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport'>org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/176/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport/org.jboss.windup.qs.rules/EjbBeanUtilsAsyncUsageRuleProviderTest/testRules/'><strong>org.jboss.windup.qs.rules.EjbBeanUtilsAsyncUsageRuleProviderTest.testRules</strong></a></li></ul>
Test FAILed.

File: config/api/src/main/java/org/jboss/windup/config/condition/GraphCondition.java [hunk 1]
```diff
@@ -62,4 +66,24 @@ public void setOutputVariablesName(String outputVariablesName)
...
         this.outputVariablesName = outputVariablesName;
     }
+
+    /**
+     * This sets the variable with the given name to the given value. If there is already a variable with the same name in the top-most stack frame,
+     * we will combine them here.
+     *
+     * This helps in the case of multiple conditions tied together with "or" or "and".
+     */
+    protected void setResults(GraphRewrite event, String variable, Iterable<? extends WindupVertexFrame> results)
```
Comment (lincolnthree) [inline config/api/src/main/java/org/jboss/windup/config/condition/GraphCondition.java]: Love it!

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/177/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/177/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport'>org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/177/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport/org.jboss.windup.qs.rules/EjbBeanUtilsAsyncUsageRuleProviderTest/testRules/'><strong>org.jboss.windup.qs.rules.EjbBeanUtilsAsyncUsageRuleProviderTest.testRules</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/811/
Test PASSed.

**Pull request merged**